sdk/spring/build-developing-version-artifacts-if-needed.md

```diff
@@ -1,28 +1,29 @@
-
-### Released version
-
-We **highly recommend** you refer to the `README.md` files of the released version.
-
-Here are the steps to get `README.md` files of the released version:
-
-1. On GitHub, navigate to the main page of the repository.
-1. Click the `master` button, then click `Tags`.
-1. In the `Find a tag` text box, input and enter the starter module name you want to search, such as `azure-spring-boot-starter-xxx`.
-1. Click one of the searched tags. (You will find that the address in your browser bar changes to that specific version. You can manually edit the address with other versions, then quickly switch to other versions.)
-1. Then check the README.md file there.
-
-
-### Developing version
+### Released version  
+  
+We **highly recommend** you refer to the `README.md` files of the released versions.  
+  
+Here are the steps to get `README.md` files of the released version:  
+  
+1 On GitHub, navigate to the main page of the repository.  
+2 Click the `master` button, then click `Tags`.   
+3 In the "Find a tag" text box, input and enter the starter module name you want to search, such as `azure-spring-boot-starter-xxx`.  
+![Search for tag](images/search-for-tag.png "Search for a tag")  
+4 Click one of the searched tags. (You will find that the address in your browser bar changes to that specific version. You can manually edit the address with other versions, then quickly switch to other versions.)  
+![Tag main page](images/tag-main-page.png "Main page of a tag")  
+5 Then check the README.md file there.  
```

Comment:
I prefer `1.` `2.` than `1` `2`.
1. `.` make it more clear, means `1` is an order, not a part of sentence.
2. It will be more convenient if we add/ delete an item.

Refs:

![image](https://user-images.githubusercontent.com/13167207/109732686-99f4df80-7bf8-11eb-8e7b-62d1e7b2a9d4.png)
![image](https://user-images.githubusercontent.com/13167207/109732735-b4c75400-7bf8-11eb-9b0f-46ea89f2a4c1.png)